Application.mk

```diff
@@ -219,7 +219,7 @@ else
 context::
 endif
 
-.depend: Makefile $(SRCS)
+.depend: Makefile $(wildcard $(SRCS))
```

Comment:
@liuguo09 @xiaoxiang781216
doesn't this break apps using VPATH/--dep-path?
my app's Make.dep is empty after this change. (as wildcard expands to empty as no source files in the current directory)

Reply:
Yes, we have to remove the generated source from Make.dep. The build sequence is:
1.context
2.depend
3.compile
The code generated dynamically need the result from phase 3(compile), but phase 2(depend) need all source code ready. This is a chicken and egg problem. We decide to not generate the dependence for the dynamically code since the dependence relationship is optinal, but the compile is critical.

Reply:
my app doesn't involve any generated code at all. everything is static. just in the other directory.

Reply:
Just update https://github.com/apache/incubator-nuttx-apps/pull/265 to fix it. Please take a look.  Thanks.